jsapp/js/projects/projectViews/projectsFieldsSelector.tsx

```diff
@@ -8,7 +8,7 @@ import KoboModalContent from '#/components/modals/koboModalContent'
 import KoboModalFooter from '#/components/modals/koboModalFooter'
 import KoboModalHeader from '#/components/modals/koboModalHeader'
 import type { ProjectFieldName } from './constants'
-import { DEFAULT_VISIBLE_FIELDS, PROJECT_FIELDS } from './constants'
+import { HOME_DEFAULT_VISIBLE_FIELDS, PROJECT_FIELDS } from './constants'
```

Comment:
Unfortunately this is a bit more complex. We have three different "project views": My Projects, My Org Projects, and custom ones. All three use the same component `UniversalProjectsRoute`. It has a prop `defaultVisibleFields` that should be passed on into `ProjectsFieldsSelector` instead of using one from `./constants`.

The fix you did fixed the problem for "My Projects", but broke it for custom views.

To have custom views, you need to set them up through admin (easiest is to create one with `*` for countries). To have org orojects views, you need to be in organization (but AFAIK, "My Org Projects" uses same defaults as "My Projects")